### ISSUE TITLE
A simple idea to improve communication 

### ISSUE THREAD

**Issue by ChiragAwale** [open-learning-exchange/open-learning-exchange.github.io#2562]
### Problem
Comments on PRs and Issues dont get read and replied as often

### Steps to reproduce the problem

N/A

### Screenshots

N/A

### Proposed solution

I think tagging the person you are replying to is a really good way which can be used along side  posting on gitter chat to get faster responses as it will help us admins/interns to efficiently find those comments, and places where responses are required.

**Comment by biswasshrestha** [comment]
I agree with this. Keeping up with the chat during busy times can be a challenge. Maybe we can make this suggestion more visible with advice on the appropriate individuals to be tagged.

**Comment by aasenomad** [comment]
I think your suggestion is going to be great because people will not just read the comment only and force them to make a reply on whatever comment their name is a tagged.

**Comment by bradlet** [comment]
@lmmrsa I would like to be assigned this issue. 

### My proposed solution:
I would like to add a note about this new convention to the **Github issues page**, or directly to the overview **first steps page** (The main one with the links to the individual steps).

This issue is likely something a lot of first step interns have issues with. I have had 3 open issues for 5 days now and, while one has gotten commented on, none have gotten the okay from an active intern to work on. 

_In my opinion,_ instantiating this as a new convention could possibly help expedite the first steps process.

**Comment by ChiragAwale** [comment]
@lmmrssa What do you think regarding @bradlet suggestion

**Comment by 007HD** [comment]
A great approach to solving this problem @bradlet

**Comment by lmmrssa** [comment]
@bradlet @007HD @ChiragAwale I think first part is good to have.

**Comment by smitshrestha101** [comment]
I think this is a great idea for us on the first steps to being an intern

**Comment by nvrqt03** [comment]
This looks like a good idea - do we know when this will be implemented?

**Comment by ChiragAwale** [comment]
@nvrqt03 It does not really need implementation in that sense. It is a way for everyone to communicate better. It is suggesting everyone do so whenever trying to communicate.

**Comment by SAI-JAHNAVI** [comment]
@bradlet @ChiragAwale Is bradlet still working on this issue or Can I help you in making the proposed changes

**Comment by spykim2003** [comment]
It's a great suggestion!

**Comment by rheam97** [comment]
@dogi i should have done this in my profile PR https://github.com/open-learning-exchange/open-learning-exchange.github.io/pull/3141#issue-1130417148
